### PR TITLE
Restore Pun Pun

### PR DESCRIPTION
**Pull request by Tayyyyyyy**
:cl: Tayyyyyyy
fix: Pun pun is back!
/:cl:

### CODE REVIEW

**Comment by tylerh95** [comment]
speed merge requested, critical bug fix

File: code/modules/mob/living/carbon/human/npcs.dm [hunk 1]
```diff
@@ -5,9 +5,8 @@
...
 
-/mob/living/carbon/human/monkey/punpun/New()
-	..()
-	spawn(1)
-		name = "Pun Pun"
-		real_name = name
-		equip_to_slot(new /obj/item/clothing/under/punpun(src), slot_w_uniform)
\ No newline at end of file
+/mob/living/carbon/human/monkey/punpun/Initialize(mapload)
+	..(mapload, "Monkey")
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/npcs.dm]: I'm not sure passing any arguments here is necessary.

Comment (Tayyyyyyy) [reply code/modules/mob/living/carbon/human/npcs.dm]: Yup, pun pun worked fine without em!

**Comment by Chronarch2954** [comment]
Pun Pun was gone?! YOU MONSTERS!

**Pull request merged**